eventmesh-protocol-plugin/eventmesh-protocol-meshmessage/src/main/java/org/apache/eventmesh/protocol/meshmessage/resolver/tcp/TcpMessageProtocolResolver.java

```diff
@@ -76,6 +76,7 @@ public static CloudEvent buildEvent(Header header, EventMeshMessage message) thr
             .withId(header.getSeq())
             .withSource(URI.create("/"))
             .withType("eventmeshmessage")
+            .withDataContentType("text/plain")
             .withSubject(topic)
             .withData(content.getBytes(StandardCharsets.UTF_8));
```

Comment:
How about replace StandardCharsets.UTF_8 with Constants.DEFAULT_CHARSET, Consistent code style.
.

Reply:
will do. thanks for your advice.

Reply:
updated.